### PR TITLE
Feature/pass profile name in devicesharedsecret

### PR DESCRIPTION
**Pull request by jkoenig134**

### CODE REVIEW

File: packages/enmeshed_types/lib/src/dtos/device_shared_secret.dart [hunk 1]
```diff
@@ -31,18 +33,14 @@ class DeviceSharedSecret extends Equatable {
     required this.username,
   });
 
-  @override
-  String toString() {
-    return 'DeviceOnboardingInfo { id: $id, createdAt: $createdAt, createdByDevice: $createdByDevice, name: $name, description: $description, secretBaseKey: $secretBaseKey, deviceIndex: $deviceIndex, synchronizationKey: $synchronizationKey, identityPrivateKey: $identityPrivateKey, identity: $identity, password: $password, username: $username }';
-  }
-
```
Comment (Milena-Czierlinski) [inline packages/enmeshed_types/lib/src/dtos/device_shared_secret.dart:38]: Unrelated fix?

Comment (jkoenig134) [reply packages/enmeshed_types/lib/src/dtos/device_shared_secret.dart:38]: I remove toStrings b/c they are useless

**Review by Milena-Czierlinski — APPROVED** [review]
